test-it/karma/utilities/TestHelpers.HttpBinHelpers.js

```diff
@@ -45,15 +45,15 @@
         request.send();
     };
 
-    var forceHttpBinQuery = function (done) {
+    var forceHttpBinQuery = function (done, relativePath) {
```

Comment:
Please put the callback at the end to match the convention of other test helpers.
`var forceHttpBinQuery = function (relativePath, done)`